web-animations-2/Overview.bs

```diff
@@ -1073,10 +1073,12 @@ an <a>animation effect</a>.
 ::  The associated <a>animation</a>'s <a>hold phase</a> if resolved, otherwise
     the <a>animation</a> timeline's [=timeline phase=].
 
-:   <dfn>at timeline boundary</dfn>
+:   <dfn>at progress timeline boundary</dfn>
 ::  Determined using the following procedure:
 
-    1.  If <em>either</em> of the following conditions are true:
+    1.  If <em>any</em> of the following conditions are true:
+        *   the associated <an>animatoin</a> does not have a
+            [=progress-based timeline=], or
```

Comment:
Trying to avoid the awkwardness of using "associated" twice in the same sentence: once for the animation being associated with the effect, and a second time for the timeline being associated with the animation.

https://drafts.csswg.org/web-animations-2/#timelines:txt=timeline%20is%20a%20progress-based%20timeline